src/DotVVM.Framework/Controls/SelectHtmlControlHelpers.cs

```diff
@@ -42,6 +42,11 @@ public static void RenderOptionsProperties(IHtmlWriter writer, SelectorBase sele
                 writer.AddKnockoutDataBind("optionsValue", "function (i) { return ko.unwrap(i)[" + KnockoutHelper.MakeStringLiteral(selector.ValueMember) + "]; }");
             }
             #pragma warning restore
+
+            if (selector.ItemTitleBinding != null)
+            {
+                writer.AddKnockoutDataBind("optionsTitle", selector.ItemTitleBinding.GetProperty<SelectorItemBindingProperty>().Expression, selector);
```

Comment:
@exyi What is the `SelectorItemBindingProperty` good for?

Reply:
@djanosik https://github.com/riganti/dotvvm/blob/master/src/DotVVM.Framework/Compilation/Binding/GeneralBindingPropertyResolvers.cs#L328 - it's a lambda of the binding.